test/sequencing.js

```diff
@@ -0,0 +1,81 @@
+import * as RA from '../src/index';
+import eq from './shared/eq';
+
+
+describe('sequencing', function() {
```

Comment:
Rename file to seq.js and use `describe('seq',...`

Reply:
Will do.